src/sentry/testutils/factories.py

```diff
@@ -303,7 +303,7 @@ def create_project_key(project):
         return project.key_set.get_or_create()[0]
 
     @staticmethod
-    def create_release(project, user=None, version=None, date_added=None):
+    def create_release(project, user=None, version=None, date_added=None, project2=None):
```

Comment:
Hmm, I would make this a list of `additional_projects`

Reply:
me too in javascript 😅need to look it up how to do that in python

Reply:
wasn't that hard 👍